client/router_logging.go

```diff
@@ -0,0 +1,115 @@
+package client
```

Comment:
Looking ahead, suggest moving this to pkg/qdr so that it could use by skupper-ce, etc. outside of the client.

Reply:
These routines are mapping between the type as exposed in the client API and the abstraction as exposed by the qdr module. Moving them into the qdr package would mean that package depended on a type that makes up part of the client API. That does not feel correct to me and I think would prevent you importing qdr without also importing the client API types.

Reply:
Agree that the type dependency is problematic, just observing that the implementation is not too coupled to the client and that skupper-ce will need similar.

Reply:
I agree with the observation. I see a couple of ways we could deal with it. One is to move the bulk of the logic for ParseRouterLogConfig (and the supporting methods), but leave the file as is. That simply reduces the amount of code to be copied. The other is to figure out how to reuse the api, or parts of it, between skupper-ce and skupper. For example we could move all this file *and* the type exposed to the api to a distinct package (e.g. log_config) and accept that package being exposed as part of the api. Thoughts?

Reply:
I would lean towards leaving as is for now, it works great. We can come back around to the client/api/pkg sharing topics as there may be other things we want to consider before making changes. Thanks.